ls1intum/Artemis
# Consistent entity annotations

### Checklist

### Motivation and Context
The annotations for some domain / entity objects were inconsistent.
All entities should use ...
```
@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

.. except there is a good reason to use something else, which should be clearly documented above the entity class in JavaDoc.

### Steps for Testing
1. Log in to Artemis
2. Navigate through the most important parts of the application while having the browser console open and check that there are no exceptions or unwanted behavior
